src/PHPStan/Faker/Reflection/FakerMethodsClassReflectionExtension.php

```diff
@@ -94,14 +164,14 @@ public function getMethod(ClassReflection $classReflection, string $methodName):
 	}
 
 	/**
-	 * @param \PHPStan\Type\Type $returnType
+	 * @param \PHPStan\Type\Type                  $returnType
 	 * @param \PHPStan\Reflection\ClassReflection $declaringClass
-	 * @param bool $static
-	 * @param bool $private
-	 * @param bool $public
-	 * @param string $name
-	 * @param mixed[] $parameters
-	 * @param bool $variadic
+	 * @param bool                                $static
+	 * @param bool                                $private
+	 * @param bool                                $public
+	 * @param string                              $name
+	 * @param mixed[]                             $parameters
+	 * @param bool                                $variadic
```

Comment:
Please, don't align parameter names and keep used code conventions.

Reply:
I fixed it.